firmware/application/apps/ui_btle_rx.cpp

```diff
@@ -70,7 +70,7 @@ BTLERxView::BTLERxView(NavigationView& nav)
     };
 
     // Auto-configure modem for LCR RX (will be removed later)
-    baseband::set_btle(persistent_memory::modem_baudrate(), 8, 0, false);
+    baseband::set_btle(37);
```

Comment:
nit: name that constant. Prefer constexpr to #define

Reply:
Done